1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,23 @@
+const mainDiv = document.querySelector('#content')
 function readingList(books) {
-  // Write your code here...
+  for (let book of books) {
+    const individualBookDiv = document.createElement('div');
+    individualBookDiv.classList.add('book-size')
```

Comment:
good descriptive names, I could follow along what you were doing just reading the code